openslides/utils/collection.py

```diff
@@ -284,8 +284,8 @@ def get_full_data(self) -> List[Dict[str, Any]]:
                 all_full_data = self.get_elements_from_db()
             else:
                 all_full_data = async_to_sync(element_cache.get_all_full_data)()
-            self.full_data = all_full_data[self.collection_string]
-        return self.full_data
+            self.full_data = all_full_data.get(self.collection_string, [])
+        return self.full_data  # type: ignore
```

Comment:
Why  do we need type: ignore? Is it possible to add the empty list case to the definition in line 273

Reply:
I think this is a mypy bug. Mypy thinks self.full_data could be None.